crates/language/src/parsers.rs

```diff
@@ -23,6 +23,9 @@ mod parser_implementation {
   pub fn language_dart() -> TSLanguage {
     tree_sitter_dart::language().into()
   }
+  pub fn language_elixir() -> TSLanguage {
+    tree_sitter_elixir::language().into()
```

Comment:
You also need to add language_elixir in `#[cfg(not(feature = "builtin-parser"))]`

See https://ast-grep.github.io/contributing/add-lang.html#add-to-ast-grep-core

Reply:
Just did, thank you.